Sources/AWSLambdaEvents/APIGateway+V2.swift

```diff
@@ -23,54 +23,122 @@ public struct APIGatewayV2Request: Codable {
             public let sourceIp: String
             public let userAgent: String
         }
-
+        
         /// Authorizer contains authorizer information for the request context.
         public struct Authorizer: Codable {
             /// JWT contains JWT authorizer information for the request context.
             public struct JWT: Codable {
                 public let claims: [String: String]?
                 public let scopes: [String]?
             }
-
+            
             public let jwt: JWT
         }
-
+        
         public let accountId: String
         public let apiId: String
         public let domainName: String
         public let domainPrefix: String
         public let stage: String
         public let requestId: String
-
+        
         public let http: HTTP
         public let authorizer: Authorizer?
-
+        
         /// The request time in format: 23/Apr/2020:11:08:18 +0000
         public let time: String
         public let timeEpoch: UInt64
     }
-
+    
     public let version: String
     public let routeKey: String
     public let rawPath: String
     public let rawQueryString: String
-
+    
     public let cookies: [String]?
     public let headers: HTTPHeaders
     public let queryStringParameters: [String: String]?
     public let pathParameters: [String: String]?
-
+    
     public let context: Context
     public let stageVariables: [String: String]?
-
+    
     public let body: String?
     public let isBase64Encoded: Bool
-
+    
     enum CodingKeys: String, CodingKey {
         case version
         case routeKey
         case rawPath
         case rawQueryString
+        
+        case cookies
+        case headers
+        case queryStringParameters
+        case pathParameters
+        
+        case context = "requestContext"
+        case stageVariables
+        
+        case body
+        case isBase64Encoded
```

Comment:
We shouldn't do this here. As stated below, create a completely separate file and type.